koku/api/provider/serializers.py

```diff
@@ -394,8 +397,9 @@ def update(self, instance, validated_data):
 
             try:
                 instance.save()
-            except IntegrityError:
-                raise serializers.ValidationError(error_obj(ProviderErrors.DUPLICATE_AUTH, message))
+            except IntegrityError as error:
+                message = f"Unknown IntegrityError: {error}"
```

Comment:
we might want to log this error and leave `message` to be the user facing string from above.  Otherwise this IntegrityError would be pushed to the sources UI

Reply:
Will we only hit IntegrityError if there is a duplicate? If that's the case, then I would rather remove this try/except altogether since I moved the raise to where we detect the duplicate.

Reply:
Yes, lets remove this try/except

Reply:
Done.